### PR TITLE
Add r-simcdm

### PR DESCRIPTION
**Pull request by coatless**
cc @conda-forge/help-r

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-simcdm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    - {{ posix }}zip               # [win]
+
+  host:
+    - llvm-openmp                  # [osx]
+    - r-base
+    - r-rcpp >=1.0.0
+    - r-rcpparmadillo >=0.9.200
+    - libblas
+    - libgomp      # [linux]
+    - liblapack
```
Comment (dbast) [inline recipes/r-simcdm/meta.yaml:42]: Looks like liblapack and llvm-openmp are maybe not required ... according to conda-build linkage analysis ... can you try removing those?

Comment (coatless) [inline recipes/r-simcdm/meta.yaml:42]: Okay, I'll remove them. Where can I see the linkage analysis? 

```suggestion
    - r-base
    - r-rcpp >=1.0.0
    - r-rcpparmadillo >=0.9.200
    - libblas
    - libgomp      # [linux]
```

File: recipes/r-simcdm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    Culpepper (2015) <doi:10.3102/1076998615595403>, and de la Torre (2009) <doi:10.3102/1076998607309474>.'
+
+  license_family: GPL-2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - coatless
```
Comment (coatless) [inline recipes/r-simcdm/meta.yaml:69]:
```suggestion
    - coatless
    - steveculpepper
```

File: recipes/r-simcdm/meta.yaml (hunk 1 above)
Comment (coatless) [reply recipes/r-simcdm/meta.yaml]: Okay, I'll remove them. Where can I see the linkage analysis?

**Comment by dbast** [comment]
@steveculpepper Do you agree being co-maintainer?

File: recipes/r-simcdm/meta.yaml (hunk 1 above)
Comment (dbast) [reply recipes/r-simcdm/meta.yaml]: You see it e.g. here https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=257853&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1327

**Comment by coatless** [comment]
@dbast for simplicity, can you accept his acknowledgement in the #13597 ?

**Comment by steveculpepper** [comment]
Y

**Comment by dbast** [comment]
@coatless Thanks

**Pull request merged**